src/main/java/de/chojo/repbot/dao/access/Analyzer.java

```diff
@@ -0,0 +1,29 @@
+package de.chojo.repbot.dao.access;
+
+import de.chojo.jdautil.util.Futures;
+import de.chojo.repbot.util.LogNotify;
+import de.chojo.sadu.base.QueryFactory;
+import org.slf4j.Logger;
+
+import javax.sql.DataSource;
+
+import static org.slf4j.LoggerFactory.getLogger;
+
+public class Analyzer extends QueryFactory {
+    private static final Logger log = getLogger(Analyzer.class);
+
+    public Analyzer(DataSource dataSource) {
+        super(dataSource);
+    }
+
+    public void cleanup() {
+        builder().query("""
+                        DELETE FROM analyzer_results WHERE analyzed < NOW() - '24 hours'::interval;
```

Comment:
In the migration we have `now() AT TIME ZONE 'utc'`, here just `now()`. Should we be consistent or can we ignore that as all entries will get deleted (maybe just some hours later)

Reply:
Nah its fine. Now returns a timestamp with a timezone. So even when we would have different timezones postgres takes this into account when comparing times. The timezone definition is more for redability inside the database table and to be consistent even when we migrate the host to another timezone.